distribution/docker/Dockerfile.viralize

```diff
@@ -1,5 +1,8 @@
+FROM opsgang/zip:latest as zipper
+
 FROM apache/druid:0.20.0
 
+COPY --from=zipper /zip /opt/zip
```

Comment:
I think you could use directly `COPY --from=opsgang/zip`, better with a pinned tag